include/wx/msw/notebook.h

```diff
@@ -192,6 +192,11 @@ class WXDLLIMPEXP_CORE wxNotebook : public wxNotebookBase
   wxPoint m_bgBrushAdj;
 #endif // wxUSE_UXTHEME
 
+private:
+      // Workaround for Windows bug to detect the correct flags for left/right
+      // tab orientation.
+      // Returns true if the point was evaluated, otherwise false.
```

Comment:
This doesn't seem right, it seems to only return false if the orientation is not horizontal.

FWIW I'd rather give it the same signature as to `HitTest()` itself and only call it when the orientation is horizontal, this would seem to be more clear.